src/main.sh

```diff
@@ -46,7 +46,7 @@ main() {
     project_id="$(teamwork::get_project_id_from_task "$task_id")"
     export TEAMWORK_PROJECT_ID=$project_id
 
-    ignored_project_ids=("$IGNORE_PROJECT_IDS")
+    ignored_project_ids=("${IGNORE_PROJECT_IDS-}")
```

Comment:
Looks good assuming it's the right syntax. For some reason I remember a colon when setting a default

Reply:
e.g. `:-`

Reply:
Yes, you're right, missing the colon there 👍